third_party/webcomponentsjs/.bower.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Can we add this as a dev_dependency instead of copying the source into our repo?